### PR TITLE
SRUM_4_Refactor_Get_Presentations

### PR DESCRIPTION
**Pull request by Jasonn5**

### CODE REVIEW

**Review by ErickaIsa — APPROVED** [review]
Clean code, good job

File: tests/presentations/conftest.py
Comment (AlisonGuzman2006) [inline tests/presentations/conftest.py:1]: The changes have been reviewed and accepted

File: tests/presentations/test_get_presentations.py
Comment (AlisonGuzman2006) [inline tests/presentations/test_get_presentations.py:1]: The changes have been reviewed and accepted

**Review by AlisonGuzman2006 — APPROVED** [review]
The changes have been reviewed and accepted

**Pull request merged**